new-lamassu-admin/src/pages/Customers/components/PhotosCard.js

```diff
@@ -0,0 +1,167 @@
+import ButtonBase from '@material-ui/core/ButtonBase'
+import Paper from '@material-ui/core/Card'
+import { makeStyles } from '@material-ui/core/styles'
+import * as R from 'ramda'
+import React, { memo, useState } from 'react'
+import Carousel from 'react-material-ui-carousel'
+
+import { InformativeDialog } from 'src/components/InformativeDialog'
+import { Info2, Label1 } from 'src/components/typography'
+import { ReactComponent as CrossedCameraIcon } from 'src/styling/icons/ID/photo/crossed-camera.svg'
+import { URI } from 'src/utils/apollo'
+
+import CopyToClipboard from '../../Transactions/CopyToClipboard'
+
+import styles from './PhotosCard.styles'
+
+const useStyles = makeStyles(styles)
+
+const Label = ({ children }) => {
+  const classes = useStyles()
+  return <Label1 className={classes.label}>{children}</Label1>
+}
+
+const PhotosCard = memo(({ frontCameraData, txPhotosData }) => {
+  const classes = useStyles()
+
+  const [photosDialog, setPhotosDialog] = useState(false)
+
+  const mapKeys = pair => {
+    const key = R.head(pair)
+    const value = R.last(pair)
+    if (key === 'txCustomerPhotoPath' || key === 'frontCameraPath') {
+      return ['path', value]
+    }
+    if (key === 'txCustomerPhotoAt' || key === 'frontCameraAt') {
+      return ['date', value]
+    }
+    return pair
+  }
+
+  const addPhotoDir = R.map(it => {
+    const hasFrontCameraData = R.has('id')(it)
+    return hasFrontCameraData
+      ? { ...it, photoDir: 'operator-data/customersphotos' }
+      : { ...it, photoDir: 'front-camera-photo' }
+  })
+
+  const standardizeKeys = R.map(
+    R.compose(R.fromPairs, R.map(mapKeys), R.toPairs)
+  )
+
+  const filterByPhotoAvailable = R.filter(
+    tx => !R.isNil(tx.date) && !R.isNil(tx.path)
+  )
+
+  const photosData = filterByPhotoAvailable(
+    addPhotoDir(standardizeKeys(R.append(frontCameraData, txPhotosData)))
+  )
+
+  const singlePhoto = R.head(photosData)
+
+  const isPhotoRollAvailable = () => {
+    return !singlePhoto
+  }
+
+  return (
+    <>
+      <Paper className={classes.photo} elevation={0}>
+        <ButtonBase
+          disabled={isPhotoRollAvailable()}
+          className={classes.button}
+          onClick={() => {
+            setPhotosDialog(true)
+          }}>
+          {singlePhoto ? (
```

Comment:
I'm not a fan of having two different boolean uses tied to the same variable. Perhaps change line 70 to `disabled={!singlePhoto}` and get rid of `isPhotoRollAvailable()`?